scripts/reboot

```diff
@@ -11,6 +11,8 @@ PLATFORM_UPDATE_REBOOT_CAUSE="platform_update_reboot_cause"
 REBOOT_CAUSE_FILE="/host/reboot-cause/reboot-cause.txt"
```

Comment:
Do you not plan to handle other kind of reboots which also arm the system with watchdog? Eg., warm-reboot?